packages/arb-token-bridge-ui/tailwind.config.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
         // Orange
         orange: '#FFEED3',
         'orange-dark': '#60461F',
+        'orange-dark-nova': '#743600',
```

Comment:
maybe move it to below `orange-arbitrum-nova` and rename it to `orange-arbitrum-nova-dark`?